wsi/drm/drmbuffer.cpp

```diff
@@ -38,8 +38,6 @@ DrmBuffer::~DrmBuffer() {
     resource_manager_->MarkResourceForDeletion(image_, image_.texture_ > 0);
   } else {
     if (image_.texture_ > 0) {
-      image_.handle_ = 0;
```

Comment:
why is this removed ?

Reply:
hmm these will be deleted twice, here and  when u delete the media handle. Any reason why u removed these lines ?